Utilities/CodeGeneration/EdFi.Ods.CodeGen/Generators/EntitiesForQueries.cs

```diff
@@ -84,95 +84,91 @@ private IEnumerable<object> GetClasses(Aggregate aggregate, Entity entity)
-                                                  }),
-                            HasOneToOnes = GetMappedNavigableOneToOnes(entity).Any(), OneToOnes = GetMappedNavigableOneToOnes(entity)
-                               .Select(
-                                    a => new
-                                         {
-                                             OtherClassName = a.OtherEntity.Name, ClassNameSuffix = QueryModelSuffix
-                                         }),
-                            NavigableChildren = GetMappedCollectionAssociations(entity)
-                                               .Where(a => _shouldRenderEntityForSchema(a.OtherEntity))
-                                               .OrderBy(a => a.Name)
-                                               .SelectMany(a => GetContextualNavigableChildren(a, classContext)),
-                            HasNonNavigableChildren = GetMappedExternalCollectionAssociations(entity)
-                               .Any(a => _shouldRenderEntityForSchema(a.OtherEntity)),
-                            NonNavigableChildren = GetMappedExternalCollectionAssociations(entity)
-                                                  .Where(a => _shouldRenderEntityForSchema(a.OtherEntity))
-                                                  .OrderBy(a => a.Name)
-                                                  .SelectMany(GetContextualNonNavigableChildren),
-                            HasNonNavigableParents = GetMappedExternalParentAssociations(entity)
-                               .Any(a => _shouldRenderEntityForSchema(a.OtherEntity)),
-                            NonNavigableParents = GetMappedExternalParentAssociations(entity)
-                                                 .Where(a => _shouldRenderEntityForSchema(a.OtherEntity))
-                                                 .OrderBy(a => a.Name)
-                                                 .Select(
-                                                      a => new
-                                                           {
-                                                               AggregateRelativeNamespace = a.OtherEntity.GetRelativeAggregateNamespace(
-                                                                   TemplateContext.SchemaProperCaseName,
-                                                                   isQueryModel: true,
-                                                                   isExtensionContext: TemplateContext.IsExtension),
-                                                               ClassName = a.OtherEntity.Name, ClassNameSuffix = QueryModelSuffix,
-                                                               AssociationName = a.Name
-                                                           })
-                        };
-                }
+                yield return
```

Comment:
can you please fix the formatting on this class while you are at it.

Reply:
You're good with the other changes to this file?

Reply:
yes, just need to reformat the file

Reply:
Done.